### PR TITLE
needinfo should not be subjected to stale state

### PR DESCRIPTION
**Pull request by yophilav**
- `needinfo` is no longer consider stale

### CODE REVIEW

File: .github/workflows/stale.yml [hunk 1]
```diff
@@ -13,4 +13,6 @@ jobs:
         stale-issue-label: 'no-issue-activity'
         days-before-stale: 30
         days-before-close: 99999
-        exempt-issue-label: 'enhancement'
+        exempt-issue-label:
```
Comment (varunpuranik) [inline .github/workflows/stale.yml]: bug as well

Comment (yophilav) [reply .github/workflows/stale.yml]: 🐛 🐞

File: .github/workflows/stale.yml [hunk 2]
```diff
@@ -13,4 +13,7 @@ jobs:
...
+          - 'enhancement'
+          - 'needinfo'
```
Comment (philipktlin) [inline .github/workflows/stale.yml]: It is correct for enhancement and bug label; but incorrect for needinfo label.

If a github issue is created and our dev mark it with "needinfo" label; then after stale period, customers doesn't provide required info, this issue should be marked as stale.

If a github issue is created, no one from our team is reviewed => no action taken; therefore even it is longer than stale period, this issue should not be marked as stale.  This case doesn't support in current github stale action.

It is good to have "bug" and "enhancement" for now.  I can see that there is more customized integration with github, e.g. enforce PR commit description.  I think we should think about an extensible solution to address them.

File: .github/workflows/stale.yml [hunk 3]
```diff
@@ -13,4 +13,7 @@ jobs:
         stale-issue-label: 'no-issue-activity'
         days-before-stale: 30
```
Comment (philipktlin) [inline .github/workflows/stale.yml]: @lt72, please confirm if you want to change days before stale from 30 to 10.  thanks.

Comment (lt72) [reply .github/workflows/stale.yml]: yes, please

**Comment by yophilav** [comment]
After some searching, it looks like the Git Stale bot does not support a conditional labeling e.g. if both label `bug` and `needinfo` are labeled, then the task can be exempted from label as a stale.

**Pull request closed without merging**